model/src/main/proto/languages.proto

```diff
@@ -30,6 +30,11 @@ enum OppiaLanguage {
 
   // Corresponds to the Swahili (Kiswahili) macro language. IETF BCP 47 language tag: sw.
   SWAHILI = 7;
```

Comment:
Just to check: are we supporting Swahili in beta too, or alpha only?

Reply:
Alpha only. The languages support for beta & GA builds are specified here: https://github.com/oppia/oppia-android/blob/develop/config/src/java/org/oppia/android/config/productionlanguages/supported_languages.textproto. The languages the app supports at all are here: https://github.com/oppia/oppia-android/blob/develop/config/src/java/org/oppia/android/config/alllanguages/supported_languages.textproto.

Reply:
Not sure if you want to merge this right now so maybe you can resolve this comment unless @seanlip gets to it first.

Reply:
Looks good. Thanks!